rjc_account/readme/USAGE.rst

```diff
@@ -1,3 +1,4 @@
 To use this module you need to:
```

Comment:
this module for modify account.
USAGE.rst not necessary to have. you can delete this file